packages/azure-themes/src/azure/styles/Slider.styles.ts

```diff
@@ -3,24 +3,94 @@ import * as StyleConstants from '../Constants';
 import { IExtendedSemanticColors } from '../IExtendedSemanticColors';
 
 export const SliderStyles = (props: ISliderStyleProps): Partial<ISliderStyles> => {
-  const { disabled, theme } = props;
+  const { disabled, theme, vertical } = props;
   const semanticColors = theme.semanticColors as IExtendedSemanticColors;
+  const sliderBoxDimension: string = '8px';
+  const sliderDiameter: string = '16px';
+  const sliderOffset: string = '2px';
+  const sliderLineBorderRadius: string = '0px';
```

Comment:
No need to have them as strings in here. `merge-styles` will treat them as px by default if a number is provided.
Also not sure, but shouldn't these be CAPITALIZED? (SLIDER_DIAMETER)

Reply:
good to know on the px.  
Happy to align to the repo convention -- would you like us to align to ALL_CAPS?

Reply:
Re: `string` vs `number` I typically favor `number` since its less prone to breakage unless you anticipate using non-pixel values here.

Reply:
Capitalize and have them have them outside the function. Thanks!

Reply:
Roger -- we'll integrate change in my next PR.  Thank you.